gate-web/src/main/groovy/com/netflix/spinnaker/gate/filters/FiatSessionFilter.groovy

```diff
@@ -61,7 +61,8 @@ class FiatSessionFilter implements Filter {
     if (fiatStatus.isEnabled() && this.enabled) {
       String user = AuthenticatedRequest.getSpinnakerUser().orElse(null)
       log.debug("Fiat session filter - found user: ${user}")
-      if (permissionEvaluator.getPermission(user) == null) {
+
+      if (user != null && permissionEvaluator.getPermission(user) == null) {
```

Comment:
This seem sensible @cfieber?

`permissionEvaluator.getPermission(user)` will return null if `user` is null, and not entirely sure about the motivations behind invalidating the session if it's found.

Pretty sure I did see a mention in the log where the user was null and the invalidation happened. Odd?